config/initializers/rack_profiler.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.env == 'development'
```

Comment:
there's a `PROFILER` section in the settings.yml.  if we were to add a flag, i'd propose adding one there, maybe `ENABLE_RACK_PROFILING`.  default to `false`, enable as needed in local settings yml files, maybe enable in the laptop and dev (and stage?) shared configs.

i'd also be totally fine with just doing it for `Rails.env == 'development'`, but the flexibility of the flag has proved beneficial for the new relic profiler as well as the ruby prof stuff that can be enabled on bulk indexing.  it's nice to be able to turn the profiling on and off at a whim for whatever instance you're debugging.